### PR TITLE
style: Guardian Directory updates

### PR DESCRIPTION
**Pull request by jcbcapps**
# SC-2682 & SC-2679

## Proposed changes
- Swap the Last Name and First Name columns
- Updates unit test for the DutyTitle column: https://github.com/USSF-ORBIT/ussf-personnel-api/pull/31

---

### Start the system
```sh
yarn services:up
yarn dev
cd ../ussf-portal-cms
yarn dev
```

Login to the portal http://localhost:3000

### Start storybook

```sh
yarn storybook
```

Login to storybook http://localhost:6006, though the command above should open it for you

---

### As the original developer, I have

- [ ] Met the acceptance criteria
- [ ] Created new stories in Storybook if applicable
- [ ] Created/modified automated unit tests in Jest
- [ ] Created/modified automated [E2E tests](https://github.com/USSF-ORBIT/ussf-portal)
- [ ] Followed guidelines for zero-downtime deploys, if applicable
- [ ] Use [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) to check for basic a11y issues

### As a reviewer, I have

Check out our [How to review a pull request](https://github.com/USSF-ORBIT/ussf-portal/blob/main/docs/how-to/review-pull-request.md) document.

---

### CODE REVIEW

**Review by shkeating — APPROVED** [review]

File: src/__tests__/pages/guardian-directory.test.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ describe('GuardianDirectory', () => {
         )
         expect(screen.getByTestId(`${idx}_Rank`)).toHaveTextContent(value.Rank)
         expect(screen.getByTestId(`${idx}_DutyTitle`)).toHaveTextContent(
-          value.DutyTitle
+          value.DutyTitle.toUpperCase()
```
Comment (gidjin) [inline src/__tests__/pages/guardian-directory.test.tsx:85]: Suggestion: I think we should handle this in the resolver for the personnel api, thoughts? https://github.com/USSF-ORBIT/ussf-personnel-api/blob/main/src/resolvers.ts

File: src/components/GuardianDirectoryTable/GuardianDirectoryTable.tsx [hunk 2]
```diff
@@ -26,7 +32,21 @@ export const GuardianDirectoryTable = ({ headers, rows, keys }: GuardianDirector
...
                 // eslint-disable-next-line security/detect-object-injection
-                return <td data-testid={`${id}_${k}`} key={`td_${k}`}>{row[k]}</td>
+                const rowValue = row[k]
+
+                if (k === 'DutyTitle') {
+                  return (
+                    <td data-testid={`${id}_${k}`} key={`td_${k}`}>
+                      {rowValue.toUpperCase()}
+                    </td>
+                  )
```
Comment (gidjin) [inline src/components/GuardianDirectoryTable/GuardianDirectoryTable.tsx:42]: Same suggestion here as above.

File: src/__tests__/pages/guardian-directory.test.tsx (hunk 1 above)
Comment (jcbcapps) [reply src/__tests__/pages/guardian-directory.test.tsx:85]: Yeah you're right, not sure why I didn't think to do that. I'll make that change and get another PR up.

Comment (jcbcapps) [reply src/__tests__/pages/guardian-directory.test.tsx:85]: Here is the PR: https://github.com/USSF-ORBIT/ussf-personnel-api/pull/30

Comment (jcbcapps) [reply src/__tests__/pages/guardian-directory.test.tsx:85]: @gidjin I removed the check on this branch for the DutyTitle column, but I left the updated unit test. I'm thinking that the PR I linked above could be merged first?

Comment (gidjin) [reply src/__tests__/pages/guardian-directory.test.tsx:85]: I'm looking at the other PR now :) thanks for the changes!

**Review by gidjin — APPROVED** [review]

**Pull request merged**